torch/utils/data/dataset.py

```diff
@@ -39,7 +39,10 @@ def __add__(self, other: 'Dataset[T_co]') -> 'ConcatDataset[T_co]':
     # See NOTE [ Lack of Default `__len__` in Python Abstract Base Classes ]
     # in pytorch/torch/utils/data/sampler.py
 
+    def on_woker_init(self) -> None:
```

Comment:
worker not woker

Reply:
Right, even though I don't think we're gonna take that exact name, I fixed the naming.